src/localization_plugin.cpp

```diff
@@ -21,7 +21,7 @@
 // ----------------------------------------------------------------------------------------------------
 
 LocalizationPlugin::LocalizationPlugin() : have_previous_pose_(false), laser_offset_initialized_(false),
-    tf_listener_(), tf_broadcaster_(0)
+    tf_listener_(0), tf_broadcaster_(0)
```

Comment:
Instead of `0`, better use `nullptr` (introduced inc++11) as it is less ambiguous in meaning. This also holds for the `tf_broadcaster_` variable.

Reply:
c++11 is not used in this repo. So lets keep it this way for now.